doc/workshops/tutorial/content/examples/tutorial5.md

```diff
@@ -0,0 +1,24 @@
+# Tutorial 5 - Plasma-Water Interface
+
+!---
+
+# Problem Statement
+
+Consider an argon DC discharge over water. Within Zapdos, users can model material surface interfaces of plasma processes (in this case, electrons diffusing into a water’s surface). This is done using what is called the Interface System within Zapdos/MOOSE.
```

Comment:
@cticenhour correct. The tutorial was design for students to change the reflection coefficient at the boundary of the water (basically, it is Alex's Zapdos paper). The Interface Kernel is discuss here since we walk through the input file with the students and show how two domains could be coupled (e.g. plasma and water).